ee/codegen_integration/test_code_to_display.py

```diff
@@ -33,4 +35,35 @@ def exclude_obj_callback(value, key):
         actual_serialized_workflow,
         exclude_obj_callback=exclude_obj_callback,
         significant_digits=6,
+        # This is for the input_variables order being out of order sometimes.
+        ignore_order=True
```

Comment:
Input variables were sometimes in the wrong order but they did have the right values so this is a quick bandaid. If this shouldn't be here then I can create a ticket to explore why the inputs display would sometimes be out of order. At first glance I assumed it was because it's a dict but that doesn't quite make sense since others don't seem to have this problem like search node